README.md

```diff
@@ -40,6 +40,11 @@ Skip automatic caching of go module directories (default: `false`)
 
 Returns the type (patch, minor, major) of the sementic version that would be required if producing a release.
 
+#### `message`
```

Comment:
nit: can we call this `output` instead of `message`? `output` seems a bit more intuitive, at least to me, because this will have the full output of the go-apidiff command.

Reply:
Done